src/antsibull_docs/jinja2/filters.py

```diff
@@ -14,6 +14,7 @@
 from collections.abc import Mapping, Sequence
 
 from antsibull_core.logging import log
+from docutils.utils import column_width as _column_width  # pyre-ignore[21]
```

Comment:
Why not just import this as `column_width` and remove the `def column_width` that just returns the result of `docutils.utils.column_width`?

Reply:
Just importing it without using it in the module makes the linters complain that this is an unused import. This way it's clear that it is used.

Reply:
Adding it to `__all__` would fix that problem, but we don't already have that set for this module, so I think this is fine. Thanks for the explanation!

/resolve